Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -839,16 +834,6 @@ private bool IsDirty
             }
         }
 
-        private bool NeedsDummyRebuild
-        {
-            get => _needsDummyRebuild;
-            set
-            {
-                _needsDummyRebuild = value;
-                _needUpdatePreview = true;
-            }
```

Comment:
this wasnt useless if it was also setting `_needUpdatePreview`, right? all the instances where it was setting this before, it should be setting `_needUpdatePreview` to true as well

Reply:
done